### DISCUSSION TITLE
LeRa? 30Pix-3000-300

### DISCUSSION THREAD

**Discussion by BlenderProxy** [TheLastBen/fast-stable-diffusion#1297] [Q&A] [answered]
Can somebody tell me what LearningRate's I must use to train my face?

30 Pics of a handsome man 
3000 Unet
LR?
300 Text
LR?

**Comment by cdefghijkl** [comment, marked as answer]
5e-6
5e-7

**Reply by BlenderProxy** [reply]
Thnx, will try. :)

**Reply by BlenderProxy** [reply]
![12361-3216513378-Comic book art of silver monocle, portrait of a middle aged grey haired man jndl with a top hat in the style of god of war, gold](https://user-images.githubusercontent.com/78314803/211399687-139bb4eb-dbb8-47e2-a752-79853147b6a8.png)
Noice :)

**Reply by BlenderProxy** [reply]
text could also be 200 I think :) have to try that out and play with it

**Reply by cdefghijkl** [reply]
Glad it works.

**Comment by TheLastBen** [comment]
for the V2, set it to 1e-5 (default), keep everything to default, use 10pics

**Reply by meunumerotim333** [reply]
What about 1.5?

**Reply by TheLastBen** [reply]
if the 1e-5 doesn't give good results, use 8e-6 and increase the unet steps
